testing/php7-memcached/APKBUILD

```diff
@@ -37,6 +37,6 @@ package() {
 	echo "extension=$_pkgreal.so" > "$pkgdir"/etc/php7/conf.d/20_$_pkgreal.ini
 }
 
-md5sums="081d43caf5a5cb4f70348280014e3da9  php7-memcached-3.0.0b1.tar.gz"
-sha256sums="a770003ebc6bb2a4976dd9ebaa9304552575c5ed9e0c998d941bdd087fee106e  php7-memcached-3.0.0b1.tar.gz"
-sha512sums="3e38885bb43aa26a69d101ad890fe73a4f545c76210450da25c067699b06c2ce7c94f900f8956ff38b5d68376bc2fadfd8c18e32a83df907a3437698c1c6ae84  php7-memcached-3.0.0b1.tar.gz"
+md5sums="081d43caf5a5cb4f70348280014e3da9  php7-memcached-$pkgver.tar.gz"
```

Comment:
No I have not, thanks for the info. I'm totally new to Apline packages, just trying to get this thing sorted out.